### DISCUSSION TITLE
Configuration check - Erasure coding settings

### DISCUSSION THREAD

**Discussion by Jazz9** [minio/minio#18693] [Q&A]
I have 6 servers, each with 12 drives.

I use MINIO_STORAGE_CLASS_STANDARD=EC:2

Would there be 4 data plus 2 parity?

I am seeing 2 drives on each server with data on it - then it is 10 data and 2 parity?

Glen

**Comment by klauspost** [comment]
@Jazz9 We have mention this many times to you before. Architecture review is only part of [subscriptions](https://min.io/pricing) and not a community service.
